src/components/FilterTabs/index.tsx

```diff
@@ -0,0 +1,75 @@
+import { FC } from "react"
+import { Button, ButtonGroup, useStyleConfig } from "@chakra-ui/react"
+import { FilterTabSize, FilterTabVariant } from "../../theme/FilterTab"
+
+export interface FilterTabProps {
+  title: string
+  tabId: string
+  onClick: (tabId: string) => void
+  selectedTabId: string
+  variant: FilterTabVariant
+  size: FilterTabSize
+}
+
+export interface FilterTabsProps {
+  variant?: FilterTabVariant
+  size?: FilterTabSize
+  selectedTabId: string
+  setSelectedTabId: (tabId: string) => void
+  tabs: {
+    title: string
+    tabId: string
+    onClick?: (tabId: string) => void
+  }[]
+}
+
+const FilterTab: FC<FilterTabProps> = ({
+  selectedTabId,
+  tabId,
+  onClick,
+  title,
+  variant,
+  size,
+}) => {
+  const isActive = selectedTabId === tabId
+  const styles = useStyleConfig("FilterTab", { isActive, variant })
+
+  return (
+    <Button
+      variant={isActive ? "outline" : "ghost"}
+      onClick={() => onClick(tabId)}
+      size={size}
+      sx={styles}
+    >
+      {title}
+    </Button>
+  )
+}
+
+export const FilterTabs: FC<FilterTabsProps> = ({
+  tabs,
+  selectedTabId,
+  setSelectedTabId,
+  variant = "primary",
+  size = "md",
+}) => {
+  const styles = useStyleConfig("FilterTabs", { variant })
+
+  return (
+    <ButtonGroup spacing="3" size={size} sx={styles}>
```

Comment:
Why are we using the `ButtonGroup` instead of `Tabs`?

Reply:
I decided to use Button group instead of Tabs because I didn't like how the `TabPanels` would work with this component. I would prefer to have the click handlers for each "tab" / button update some state however they feel is needed, rather than having control over a TabPanel. A good example is the `Mint/Umint` filter, which will update some state in the tbtc context that the entire page needs access to. Lmk what you think